pkg/cluster/logs/logs.go

```diff
@@ -113,9 +125,13 @@ func Collect(nodes []nodes.Node, dir string) error {
 	return concurrent.Coalesce(fns...)
 }
 
-// untar reads the tar file from r and writes it into dir.
-func untar(r io.Reader, dir string) (err error) {
-	tr := tar.NewReader(r)
+// untargz reads the tar compressed file from r and writes it into dir.
+func untargz(r io.Reader, dir string) (err error) {
```

Comment:
is this actually more performant? splitting this PR up

Reply:
it reduces the amount of bytes going through the pipe, the performance is the same but I think we are less exposed to the docker exec thing

Reply:
it does reduce the amount of bytes but I doubt the peformance is actually the same given we have to both compress and decompress, there's going to be a tradeoff in the resources used for that and the resources used to push bytes over the exec stream

Reply:
check last comments with a comparison of both

Reply:
removing gzip because is not adding any benefit